Reworked boss timer a bit and many things

AzuredBlue opened the pull request:
+ Fixed Terracotta time in M6 (maybe)
+ Improved Simon Says Solver
+ Fixed noChildLeftBehind (GUI name is different)
+ Reworked M7 Boss Timer (Added a Terminals Split, changed messages to more accepted split messages)
+ Allowed to change box starred mobs color
+ Improved prevent cursor reset to not always reset
= Hide Tooltip in Melody Terminal instead of Maze (Good night maze, not coming back)
= Fixed legion display being wrong sometimes
 - Removed Portal Entry Timer and Hide Pet Nametags (Portal is now instant and you can hide pet nametags in pet settings)

AzuredBlue committed:
many things

fixed terracotta time maybe
fixed noChildLeftBehind
removed portal entry timer and hide pet nametags
reworked m7 boss timer
replaced maze with melody when hiding the tooltip

AzuredBlue committed:
Improved prevent cursor reset (shoutout bonsai)

AzuredBlue committed:
Allowed to change box starred mobs color

My-Name-Is-Jeff commented on src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/PetFeatures.kt:
```diff
@@ -53,22 +53,6 @@ object PetFeatures {
     private var lastPetLockNotif: Long = 0
     var lastPet: String? = null
 
-
```
uhhh I can think of a case where this is useful, cause maybe hypixel hide pet nametags might mess up with some mod feature or something idek

AzuredBlue replied:
doubt a mod uses pet nametags for anything since most people have pets hidden anyways

My-Name-Is-Jeff commented on src/main/java/gg/skytils/skytilsmod/mixins/transformers/util/MixinMouseHelper.java:
```diff
@@ -18,16 +18,21 @@
 
 @Mixin(MouseHelper.class)
 public class MixinMouseHelper {
-    @WrapWithCondition(method = "ungrabMouseCursor", at = @At(value = "INVOKE", target = "Lorg/lwjgl/input/Mouse;setCursorPosition(II)V", remap = false))
```
I'm pretty sure wrap with condition is better because if other modifications are made it still works

AzuredBlue replied:
> I'm pretty sure wrap with condition is better because if other modifications are made it still works

no clue how wrap with condition works

AzuredBlue committed:
Improved A LOT the Simon Says Solver

Also made trapper thing only work in the farming island

AzuredBlue committed:
Fixed legion display

Watchdog has UUID v4, nicked players have uuid v1

AzuredBlue committed:
Reworked Visitor Notification

Now works with chat messages and an essentials notification.
Also pretty sure it didnt work when you went offline

AzuredBlue commented:
i think its done
making simon says boxes more accurately cover the button would be nice but thats too much effort
same with garden features

AzuredBlue committed:
Improved the box for simon says buttons

My-Name-Is-Jeff committed:
simplify code on simon says rework

My-Name-Is-Jeff committed:
Revert changes to MixinMouseHelper

Remove unneeded variable guiWasNotNull
